src/styles/_global-styles.scss

```diff
@@ -42,6 +46,19 @@ ds-admin-sidebar {
     z-index: 0;
 }
 
+.klaro
+.cookie-notice
+.cm-btn.cm-btn-success {
+    color: #333333 !important;
+    background-color: #4de0b4 !important;
+}
+
+.klaro
+.cookie-notice
+a{
+    color: #4de0b4 !important;
+}
+
```

Comment:
In your code, the usage of !important is unnecessary. Instead, you can enhance the specificity of your selectors and leverage SCSS's features to avoid duplicating selectors. Here's an example of how your code can be refactored to eliminate the need for !important:

```
.klaro,
.cookie-notice,
.cn-buttons {
  .cm-btn.cm-btn-success {
    color: #333333;
    background-color: #4de0b4;
  }

  a {
    color: #4de0b4;
  }
}
```
Additionally, it seems you've deviated from the approach of using new variables. The variable `--green1` is used in the Klaro class and it works perfectly. I believe the original intention was to use this variable for maintainability and consistency. Here's how the code should look with the variable:

```
:root {
   --green1: #4de0b4;  // This variable represents the success color for the Klaro cookie banner
   --button-text-color: #333; // This variable represents the text color for buttons in the Klaro cookie banner
}

.klaro,
.cookie-notice,
.cn-buttons {
  .cm-btn.cm-btn-success {
    color: var(--button-text-color);
    background-color: var(--green1);
  }

  a {
    color: var(--green1);
  }
}
```